sdk/servicebus/azure-servicebus/tests/perf_tests/receive_queue_message_stream.py

```diff
@@ -23,9 +23,9 @@ def run_sync(self):
                     break
                 count += 1
 
-    async def run_async(self):
+    async def run_async(self) -> None:
         count = 0
-        if self.args.peeklock:
+        if not self.args.peeklock:
```

Comment:
This loop is running `complete_message`, so it _is_ peeklock isn't it?

Reply:
ah I got mixed up on receive & delete vs peeklock and complete. Has to be fixed across all the tests